x-pack/plugins/alerting/server/routes/lib/actions_schema.ts

```diff
@@ -24,6 +24,7 @@ export const actionsSchema = schema.arrayOf(
         throttle: schema.nullable(schema.string({ validate: validateDurationSchema })),
       })
     ),
+    uuid: schema.maybe(schema.string()),
```

Comment:
uuid is required in a ruleAction. We will use it to track throttled actions.
Create and Update APIs uses it if you send, otherwise generate one.
So the only required thing from security-solutions is sending back the actions with uuid on update, that's it :)